contrib/terraform/terraform.py

```diff
@@ -318,7 +318,9 @@ def openstack_host(resource, module_name):
 
     # attrs specific to Ansible
     if 'metadata.ssh_user' in raw_attrs:
-        attrs['ansible_ssh_user'] = raw_attrs['metadata.ssh_user']
+        attrs['ansible_user'] = raw_attrs['metadata.ssh_user']
+    if 'metadata.ssh_port' in raw_attrs:
+        attrs['ansible_port'] = raw_attrs['metadata.ssh_port']
```

Comment:
Do we still need `'ansible_ssh_port': 22,` of line 288?

Reply:
nope, it has been removed here https://github.com/kubernetes-sigs/kubespray/pull/9828

Reply:
omg, we have already done :-)

/ok-to-test
/approve